### PR TITLE
Fixed Line Plot dialog translation layout

### PR DESCRIPTION
**Pull request by Fidel365**
Fixes (partially) #7650 
fixes translation layout in french
Describe>Specific Tables/Graphs>Line/Smooth/Dumbbell/Slope plot....
@N-thony , @rdstern , this PR is ready for review

### CODE REVIEW

File: instat/dlgLinePlot.vb [hunk 1]
```diff
@@ -730,4 +730,12 @@ Public Class dlgLinePlot
...
         End If
     End Sub
+
+    Private Sub AllControl_ControlContentsChanged(ucrChangedControl As ucrCore) Handles ucrVariablesAsFactorForLinePlot.ControlContentsChanged, ucrSave.ControlContentsChanged, ucrReceiverXEnd.ControlContentsChanged, ucrReceiverX.ControlContentsChanged, ucrReceiverSlopeY.ControlContentsChanged, ucrReceiverSlopeX.ControlContentsChanged, ucrReceiverSlopeColour.ControlContentsChanged, ucrFactorOptionalReceiver.ControlContentsChanged
+
+    End Sub
+
+    Private Sub UcrVariablesAsFactor_ControlValueChanged(ucrChangedControl As ucrCore) Handles ucrVariablesAsFactorForLinePlot.ControlValueChanged
+
+    End Sub
```
Comment (lloyddewit) [inline instat/dlgLinePlot.vb:740]: Remove empty subs?

**Review by lloyddewit — CHANGES_REQUESTED** [review]

**Comment by Fidel365** [comment]
@lloyddewit , I have made the changes requested

**Review by lloyddewit — APPROVED** [review]

**Comment by lloyddewit** [comment]
@rdstern if you can test/approve, then we can merge, thanks

**Comment by N-thony** [comment]
@Fidel365 please align the controls well
image
image
image
image

**Comment by Fidel365** [comment]
@N-thony , I have made the changes requested

File: instat/dlgLinePlot.vb [hunk 2]
```diff
@@ -730,4 +730,8 @@ Public Class dlgLinePlot
             clsOptionsFunction.RemoveParameterByName("formula")
         End If
     End Sub
+
+    Private Sub ucrInputFormula_Load(sender As Object, e As EventArgs) Handles ucrInputFormula.Load
+
+    End Sub
```
Comment (N-thony) [inline instat/dlgLinePlot.vb:736]: can you remove this?

**Review by N-thony — CHANGES_REQUESTED** [review]

**Comment by Fidel365** [comment]
@N-thony , I have made the changes

**Comment by N-thony** [comment]
@Fidel365 could we have a quick call?

**Comment by Fidel365** [comment]
@N-thony , yes offcourse

**Comment by Fidel365** [comment]
@N-thony , this PR is ready for rereview

**Review by N-thony — APPROVED** [review]
@rdstern over to you

**Review by lloyddewit — APPROVED** [review]

**Comment by lloyddewit** [comment]
@rdstern If you can test/approve, then we can merge, thanks

**Comment by rdstern** [comment]
@Fidel365 this is looking quite good now.  
image

I notice the Add SE is just Ajouter.  Perhaps @N-thony can advise if it could become just a bit wider and be Ajouter ES, or es, or we need to have erreur standard?
Then the line Utiliser le chemin ou - I am not sure about, because it remains in French when we click on the Lan button.  I suggest here we change the English and delete the "Use".  So in English it just says Path or Step.  Then it will be wide enough in French.  And we should change it in the radio buttons too.

Maybe this can be done quickly now by @N-thony or @MeSophie?

**Comment by MeSophie** [comment]
> @Fidel365 this is looking quite good now. image
> 
> I notice the Add SE is just Ajouter. Perhaps @N-thony can advise if it could become just a bit wider and be Ajouter ES, or es, or we need to have erreur standard? Then the line Utiliser le chemin ou - I am not sure about, because it remains in French when we click on the Lan button. I suggest here we change the English and delete the "Use". So in English it just says Path or Step. Then it will be wide enough in French. And we should change it in the radio buttons too.
> 
> Maybe this can be done quickly now by @N-thony or @MeSophie?

@rdstern the modifications that I made are on PR#7951. And also changed **Ajouter SE** to **Ajouter ES** in Crowdin.

**Comment by lloyddewit** [comment]
@rdstern I approved the 'Ajouter ES' change in CrowdIn. It will be visible when PR #7943 is merged. Are you are ready to approve this PR (PR #7762) based on @MeSophie 's comment above? If so, then we can merge, thanks

**Review by rdstern — APPROVED** [review]
Absolutely!

**Pull request merged**